libraries/shared/src/shared/Camera.h

```diff
@@ -110,6 +113,36 @@ public slots:
      */
     void setOrientation(const glm::quat& orientation);
 
+    /**jsdoc
+     * Gets the current mouse capture state.
+     * @function Camera.getMouseCapture
+     * @returns {boolean} The current mouse capture state.
+     */
+    bool getCaptureMouse() const { return _captureMouse; }
+
+    /**jsdoc
+     * Sets the mouse capture state.  When <code>true</code>, the mouse will be invisible and cannot leave the bounds of
+     * Interface, as long as Interface is the active window and no menu item is selected.  When <code>false</code>, the mouse
+     * will behave normally.
+     * @function Camera.setCaptureMouse
+     * @param {boolean} captureMouse - <code>true</code> to capture the mouse, <code>false</code> to release the mouse.
+     */
+    void setCaptureMouse(bool captureMouse) { _captureMouse = captureMouse; emit captureMouseUpdated(captureMouse); }
+
+    /**jsdoc
+     * Gets the current camera sensitivity.
+     * @function Camera.getSensitivity
+     * @returns {boolean} The current camera sensitivity.
+     */
+    float getSensitivity() const { return _sensitivity; }
+
+    /**jsdoc
+     * Sets the camera sensitivity.  Higher values mean that the camera will be more sensitive to mouse movements.
+     * @function Camera.setSensitivity
+     * @param {boolean} sensitivity - The desired camera sensitivity.
+     */
+    void setSensitivity(float sensitivity) { _sensitivity = glm::max(0.0f, sensitivity); }
```

Comment:
Should there be an upper bound to the sensitivity value?

Reply:
I think not needed for now.  In the Preferences dialog it maxes out at 5, but no need to restrict people